This is a testharness.js-based test.
FAIL navigator.getBattery() shall throw a 'SecurityError' in an insecure context assert_unreached: Should have rejected: undefined Reached unreachable code
Harness: the test ran to completion.

